src/content/en/2022/privacy.md

```diff
@@ -1,22 +1,466 @@
 ---
-#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Privacy
-#TODO - Review and update chapter description
 description: Privacy chapter of the 2022 Web Almanac covering adoption and impact of online tracking, privacy preference signals and browser initiatives for a privacy-friendlier web.
-authors: [tomvangoethem, spanicker, nrllh]
-reviewers: [miketaylr, iskander-sanchez-rola, SaptakS]
+authors: [tomvangoethem, nrllh]
+reviewers: [iskander-sanchez-rola]
```

Comment:
We should add Melissa Knight as a reviewer as well, I see lots of feedback from them in the doc.